apps/web/src/lang/fr.json

```diff
@@ -206,6 +206,10 @@
     "defaultMessage": "Lier des compétences en vedette",
     "description": "Title for skills on Experience form"
   },
+  "/IMv2G": {
+    "defaultMessage": "Prochainement",
```

Comment:
`q5FQbu`, **Coming Soon!** is translated as _À venir!_. 
`/IMv2G`, **Coming soon** is translated as _Prochainement_.

Which one should be used @gobyrne or are both correct and should be left?

Reply:
Updated to use "À venir" (no punctuation as English string also omits it) [update coming soon translation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8004/commits/b9f44749eb44355023fcbc51aa9151add13a3ee6)